contentcuration/contentcuration/frontend/channelEdit/views/TreeView/index.vue

```diff
@@ -1,78 +1,75 @@
 <template>
 
-  <VContainer fluid class="pa-0">
-    <Banner
-      v-model="hasStagingTree"
-      border
-      data-test="staging-tree-banner"
-    >
-      <VLayout align-center justify-start>
-        <Icon>build</Icon>
-        <span class="pl-1">
-          <!--
-            v-if="hasStagingTree" to prevent the link from being rendered
-            when banner is hidden because there is no staging tree
-          -->
-          <router-link
-            v-if="hasStagingTree"
-            :to="stagingTreeLink"
-            :style="{'text-decoration': 'underline'}"
-            data-test="staging-tree-link"
-          >{{ $tr('updatedResourcesReadyForReview') }}</router-link>
-          (<time :datetime="channelModifiedDate">{{ prettyChannelModifiedDate }}</time>)
-        </span>
-      </VLayout>
-    </Banner>
-    <VLayout row>
-      <ResizableNavigationDrawer
-        v-show="!isEmptyChannel"
-        ref="hierarchy"
-        permanent
-        clipped
-        localName="topic-tree"
-        class="hidden-xs-only"
-        :maxWidth="400"
-        :minWidth="200"
-        :style="{backgroundColor: $vuetify.theme.backgroundColor}"
+  <TreeViewBase>
+    <VContainer fluid class="pa-0">
+      <Banner
+        v-model="hasStagingTree"
+        border
+        data-test="staging-tree-banner"
       >
-        <VLayout row>
-          <IconButton
-            icon="collapse_all"
-            :text="$tr('collapseAllButton')"
-            @click="collapseAll"
-          >
-            $vuetify.icons.collapse_all
-          </IconButton>
-          <VSpacer />
-          <IconButton
-            :disabled="!ancestors || !ancestors.length"
-            icon="gps_fixed"
-            :text="$tr('openCurrentLocationButton')"
-            @click="jumpToLocation"
-          />
+        <VLayout align-center justify-start>
+          <Icon>build</Icon>
+          <span class="pl-1">
+            <!--
+              v-if="hasStagingTree" to prevent the link from being rendered
+              when banner is hidden because there is no staging tree
+            -->
+            <router-link
+              v-if="hasStagingTree"
+              :to="stagingTreeLink"
+              :style="{'text-decoration': 'underline'}"
+              data-test="staging-tree-link"
+            >{{ $tr('updatedResourcesReadyForReview') }}</router-link>
+            (<time :datetime="channelModifiedDate">{{ prettyChannelModifiedDate }}</time>)
```

Comment:
Will need to figure out a different way of handling this to support i18n

Reply:
@jayoshih I am not sure what's meant by this comment exactly, we're using `vue-intl`, isn't that sufficient?